MAUI/TreeView/Item-height-customization.md

```diff
@@ -0,0 +1,134 @@
+       <local:FileManagerViewModel x:Name="viewModel"></local:FileManagerViewModel>
+    </ContentPage.BindingContext>
+    <ContentPage.Content>
+       <syncfusion:SfTreeView x:Name="treeView"
+                              QueryNodeSize="TreeView_QueryNodeSize"
+                              ChildPropertyName="SubFiles"
+                              ItemsSource="{Binding ImageNodeInfo}"/>
+       </syncfusion:SfTreeView>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+public class MainPage : ContentPage
+{
+    public MainPage()
+    {
+      InitializeComponent();
+      this.treeView.QueryNodeSize += TreeView_QueryNodeSize;
+    }
+    
+    private void TreeView_QueryNodeSize(object sender, QueryNodeSizeEventArgs e)
+    {
+        if (e.Node.Level == 0)
+        {
+            //Returns speified item height for items.
+            e.Height = 200;
+            e.Handled = true;
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Customize specific item height based on the content size
+
+The TreeView allows adjusting height of items based on the content measured size while loaded by setting the `Height` argument with value returned from `QueryNodeSizeEventArgs.GetActualNodeHeight` method.
+                                             
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.TreeView;assembly=Syncfusion.Maui.TreeView"
+             xmlns:local="clr-namespace:GettingStarted"
+             x:Class="GettingStarted.MainPage">
+    <ContentPage.BindingContext>
+       <local:FileManagerViewModel x:Name="viewModel"></local:FileManagerViewModel>
+    </ContentPage.BindingContext>
+    <ContentPage.Content>
+       <syncfusion:SfTreeView x:Name="treeView"
+                              QueryNodeSize="TreeView_QueryNodeSize"
+                              ChildPropertyName="SubFiles"
+                              ItemsSource="{Binding ImageNodeInfo}"/>
+       </syncfusion:SfTreeView>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+public class MainPage : ContentPage
+{
+    public MainPage()
+    {
+      InitializeComponent();
+      this.treeView.QueryNodeSize += TreeView_QueryNodeSize;
+    }
+    
+    private void TreeView_QueryNodeSize(object sender, QueryNodeSizeEventArgs e)
+    {
+        if (e.Node.Level != 0)
+        {
+            // Returns item height based on the content loaded.
+            e.Height = e.GetActualNodeHeight();
+            e.Handled = true;
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+## Limitations
+
+ * Define the size of the image when loading image in the `SfTreeView.ItemTemplate`. Because, it does not return actual measured size when measuring before item layout.

```

Comment:
* Define the size of the image when loading an image in the `SfTreeView.ItemTemplate`. Because it does not return the actual measured size when measuring before the item layout.